pkg/v2/json/deserialize.go

```diff
@@ -403,6 +403,18 @@ func (d *deserializeState) parseBooleanProperty() error {
 		if _, err := d.navigator.Current().Replace(false); err != nil {
 			return err
 		}
+	} else if p.Attribute().Path() == "active" {
+		// Hack to work around Azure AD bugs
+		// they send "True" for active instead of true
+		if d.isTrueInQuotes(start, end) {
+			if _, err := d.navigator.Current().Replace(true); err != nil {
+				return err
+			}
+		} else if d.isFalseInQuotes(start, end) {
+			if _, err := d.navigator.Current().Replace(false); err != nil {
+				return err
+			}
+		}
```

Comment:
I understand this is just a hack fix for Microsoft AD. I am happy to include that in the code. But I would like to do that in a more cautious way so that readers can immediately understand this is a hack.

I will merge this in on a `fix/67` branch and move it down into the `else` clause, so people are clear this is not _among_ the usual true/false case. And I will give it a more hack-obvious name.